helm_deploy/hmpps-assessments-api/values.yaml

```diff
@@ -49,8 +49,8 @@ generic-service:
       DATABASE_NAME: "database_name"
       DATABASE_ENDPOINT: "rds_instance_endpoint"
     hmpps-assessments-elasticache-redis:
-      SPRING_REDIS_HOST: "primary_endpoint_address"
-      SPRING_REDIS_PASSWORD: "auth_token"
+      SPRING_DATA_REDIS_HOST: "primary_endpoint_address"
```

Comment:
you've missed changing the `SPRING_REDIS_SSL` property higher up in this file

Reply:
Fixed